lib/css/components/_stacks-tables.less

```diff
@@ -304,6 +304,17 @@
         padding-left: 0;
         width: 120px;
     }
+
+    //  $$  Disabled rows
+    //  ------------------------------------------------------------------------
+    tr.is-disabled {
+        background-color: @black-025;
+
+        th:not(.never-disabled),
```

Comment:
@aaronshekey I'm not happy about defining `<th>` and `<td>` separately. `:not` and `*:not` didn't work. However can `<tr>`'s have any other children other than `<th>` and `<td>`? I don't think so. I also can't think of another element that we'd want to apply this class to (eg. specific elements _within_ a table cell? Nah).